src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -332,8 +332,18 @@ Result AudioStreamAAudio::open() {
 
 error2:
     mLibLoader->builder_delete(aaudioBuilder);
-    LOGD("AudioStreamAAudio.open: AAudioStream_Open() returned %s",
-         mLibLoader->convertResultToText(static_cast<aaudio_result_t>(result)));
+    LOGD("AudioStreamAAudio.open: AAudioStream_Open() returned %s = %d",
+         mLibLoader->convertResultToText(static_cast<aaudio_result_t>(result)),
+         static_cast<int>(result));
+    if (static_cast<int>(result) > 0) {
+        // Possibly due to b/267531411
+        if (OboeGlobals::areWorkaroundsEnabled()) {
+            result = Result::ErrorInternal; // Coerce to negative error.
+        } else {
+            LOGW("AudioStreamAAudio.open: AAudioStream_Open() returned positive error = %d",
```

Comment:
We should add this warning message for both cases for easier debugging

Reply:
Done